app/services/replication_transfer_updater.rb

```diff
@@ -83,7 +83,7 @@ def request_storage
       if params[:fixity_value] == correct_fixity
         record.update(store_requested: true)
       else
-        record.cancel!("fixity_reject")
+        record.cancel!("fixity_reject", params[:cancel_reason_detail])
       end
```

Comment:
I ended up deciding to just say what we got and what we expected.  For now both of these values are in the registry anyway.  In the future, publishing the nonced fixity forces discarding the nonce and actually provides something meaningful the to_node can use out of band to try to decipher what went wrong.